usage-guides/assets.md

```diff
@@ -17,6 +17,28 @@ To solve this problem we added a whitelist which you can add assets to as can be
 
 ![Whitelist asset](/images/asset_whitelist_menu.png)
 
+## Whitelisting and Re-detecting Missing Tokens
+
+If a token doesn't appear in your balances:
+
+### Check Asset Status
+
+Go to the Manage Assets -> Assets page
+Use the search to find the token by name or address
+Verify if the token is ignored
+If ignored, click to whitelist it
+
+![Example of managing assets and whitelisting a token in the Manage Assets page.](/images/manage-assets-refresh.png)
+
+### Re-detect After Whitelisting
+
+Go to Account & Balances -> Blockchain Balances
+Scroll down to EVM accounts and click on Re-detect tokens (🔃) Here you can specify chains for re-detection.
+![Example of re-detecting tokens for all EVM chains in the Blockchain Balances section.](/images/redetect-tokens-chain.png)
+
+If still not appearing, try the full token detection from the dashboard by clicking on 🔃 toggle.
+![Example of performing a full token detection using the re-detect toggle on the dashboard.](/images/redetect-tokens-dashboard.png)
+
```

Comment:
In the last commit you seem to have completely removed the part you added in the first commit which was what I asked you to make this PR for. Explaining the common problem that people have with accidentally ignoring assets or having it flagged by the spam detection method.

Reply:
I just checked, for this part, it has been added here https://docs.rotki.com/usage-guides/troubleshooting.html#my-asset-is-not-showing-in-rotki

Reply:
@cleanerzkp Can you check if the information is duplicated with this one https://docs.rotki.com/usage-guides/troubleshooting.html#my-asset-is-not-showing-in-rotki , or perhaps if you need to put the link to there